.github/workflows/checklist_actions.yml

```diff
@@ -41,7 +41,7 @@ jobs:
   
   frontend_pr_commented:
     name: Frontend PR comment
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == 'add_frontend_checklist' }}
+    if: ${{ github.event.issue.pull_request }}
```

Comment:
I don't think you should push changes in this unrelated files. So can you revert it?